cmd/chronograf-migrator/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/influxdata/influxdb/chronograf"
+	"github.com/influxdata/influxdb/chronograf/bolt"
+	"github.com/influxdata/influxdb/pkger"
+	"gopkg.in/yaml.v3"
+)
+
+var chronografDBPath string
+var outputFile string
+
+func init() {
+	flag.StringVar(&chronografDBPath, "db", "", "path to the chronograf database")
```

Comment:
might as well axe the `init` funcs and throw the flags in before the parse call 🤷‍♂, food for thought from an anti initer 😬